src/pages/index.js

```diff
@@ -211,13 +211,15 @@ function handleAvatarFormSubmit(data) {
 
 profileEditButton.addEventListener("click", () => {
   profileEditForm.open();
+  editFormValidator.resetValidation();
   const userData = userInfo.getUserInfo();
   profileTitleInput.value = userData.title;
   profileDescriptionInput.value = userData.description;
 });
 
 addNewCardButton.addEventListener("click", () => {
   addCardPopup.open();
+  addCardValidator.resetValidation();
```

Comment:
`resetValidation` is for clearing errors but you shouldn't call it here because you don't reset the form on opening

Reply:
Deleted line 222 as well. Not currently getting any errors. Hope this can get approved now. Thanks.